ui/StatusQ/src/StatusQ/Controls/StatusComboBox.qml

```diff
@@ -157,6 +157,7 @@ Item {
                     model: comboBox.popup.visible ? comboBox.delegateModel : null
                     currentIndex: comboBox.highlightedIndex
                 }
+                onClosed: { destroy(); }
```

Comment:
Not sure it's desired here; it's part of the component, there's no need to destroy this if we hadn't created it explicitly ourselves